flexprep/config/aviso/listener_diss.yaml

```diff
@@ -0,0 +1,25 @@
+listeners:
+  - event: dissemination
+    request:
+      destination: S7Y
+      class: od
+      expver: 0001
+      domain: g
+      stream: oper
+    triggers:
+      - type: command
+        working_dir: "$HOME"
+        command: >
+          docker run \
+            --mount type=bind,source=$HOME/.sqlite/,destination=/src/app-root/db/
+            --env-file .env \
+            493666016161.dkr.ecr.eu-central-2.amazonaws.com/numericalweatherpredictions/flexpart_ifs/flexprep:2408.8a89852bd0a76e1930ea53b992c5035423d91485 \
```

Comment:
The image tag will be updated because once this is merged, I will deploy it and update this tag.